internal/hmyapi/apiv1/transactionpool.go

```diff
@@ -306,11 +356,11 @@ func (s *PublicTransactionPoolAPI) GetTransactionReceipt(ctx context.Context, ha
 	receipt := receipts[index]
 	fields := map[string]interface{}{
 		"blockHash":         blockHash,
-		"blockNumber":       hexutil.Uint64(blockNumber),
+		"blockNumber":       blockNumber,
```

Comment:
I was thinking that apiv1 is still suppose to use hex values and apiv2 will use decimals. If so, this is inconsistent right?

Reply:
ah i see. I wasn't aware of that. Reverted the changes in transactionpool apiv1.